content/en/01-about-pixie/03-pixie-ebpf.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "Pixie + eBPF"
+metaTitle: "About Pixie | Pixie + eBPF"
+metaDescription: "How Pixie uses eBPF to automatically collect data"
+order: 3
+---
+
+Pixie uses eBPF to drive much of the data collection on your Kubernetes applications.
+Pixie's eBPF based approach enables it to collect the data in a way that requires no user instrumentation
+(i.e. no code changes, no redeployments), and with very low overhead.
+
+Pixie sets up eBPF probes to trigger on a number of kernel or user-space events,
```

Comment:
```suggestion
During deployment, Pixie sets up eBPF probes to trigger on a number of kernel or user-space events
```

Reply:
1) Not entirely true that we only set up eBPF probes during deployment. We also add probes at other times as well.
2) The "During deployment" makes the "at which time it collects..." a bit ambiguous.

Reply:
Is this second paragraph necessary? I think it may duplicate information below.